Report/Properties/SharedAssemblyInfo.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Reflection;
+
+// common assembly attributes
+[assembly: AssemblyDescription("Lean Engine is an open-source, platform agnostic C# and Python algorithmic trading engine. " +
```

Comment:
This file should be shared with the `Common` project, can see examples of how in the Launcher project